src/components/Categories/Categories.jsx

```diff
@@ -1,9 +1,17 @@
+import { useDispatch, useSelector } from 'react-redux';
 import './Categories.css';
+import { statusAction, selectCategories } from '../../redux/categories/categories';
 
-const Categories = () => (
-  <div className="categories">
-    <button type="button">Check status</button>
-  </div>
-);
+const Categories = () => {
+  const dispatch = useDispatch();
+  const cat = useSelector(selectCategories);
+
+  return (
+    <div className="categories">
+      {cat.length ? <p>{cat}</p> : ''}
+      <button type="button" onClick={() => dispatch(statusAction())}>Check status</button>
```

Comment:
- [ ] [optional] @Shisui6 great work in implementing the changes requested, it would be nice if you created a method to add the functionality instead of calling the dispatch method in your return section.